### DISCUSSION TITLE
0.27.6 是最终版本了?

### DISCUSSION THREAD

**Discussion by heeroyuyj** [1715173329/UnblockNeteaseMusic#1375] [General]
看 release 里写道

> So, this might be the last release due to the lack of maintenance, consider migrating to Spotify, we use this too.
Of course, feel free to take over the maintainership if you are interested in it :-)

以后不再支持网易云音乐了?

**Comment by pan93412** [comment]
不是这样，只是我们会主要以合并 PR、维持现状为主。你有什么建议也是可以继续提出，只是往后不会再这么频繁的发版本、加功能了。

**Reply by ITManCHINA** [reply]
> 不是这样，只是我们会主要以合并 PR、维持现状为主。你有什么建议也是可以继续提出，只是往后不会再这么频繁的发版本、加功能了。

请问一下，最近有更新新接口的方案或者计划嘛？没有版权的日子好难过🥲
